plugins/notifications-backend/package.json

```diff
@@ -21,18 +21,23 @@
     "clean": "backstage-cli package clean",
     "prepack": "backstage-cli package prepack",
     "postpack": "backstage-cli package postpack",
-    "tsc": "tsc"
+    "tsc": "tsc",
+    "openapi": "npx openapicmd typegen src/openapi.yaml > src/openapi.d.ts"
```

Comment:
Without it it does not work

Reply:
This is not a proper way of calling it, most probably there is a left over from your previous trying or a global dependency. If we need the `openapicmd`, we should change the script to
```
"openapi": "openapi typegen src/openapi.yaml > src/openapi.d.ts"
```

and run
```
cd plugins/notifications-backend
yarn add openapicmd -D
```

This will add the `opencmd` to the node_modules.
And the `plugins/notifications-backend/node_modules/.bin/openapi` will become a symlink to the `openapicmd` package.
Don't ask me why they use `openapi` to call `openapicmd`. 
But for sure, we need to add the `openapicmd` among our dependencies if we want to use it.

Reply:
fixed